cmd/launcher/doctor.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func runDoctor(args []string) error {
+	attachConsole()
```

Comment:
This ignores the returned error. I think it's reasonable to continue execution, but we should either log the error here, or log it in `attachConsole` and return nothing

Reply:
Consolidating discussion to https://github.com/kolide/launcher/pull/1438#discussion_r1382079870 🙂